@xen-orchestra/fs/src/s3.js

```diff
@@ -246,7 +246,7 @@ export default class S3Handler extends RemoteHandlerAbstract {
         Prefix: this._dir + path + '/',
         ContinuationToken: NextContinuationToken,
       })
-      NextContinuationToken = result.isTruncated ? null : result.NextContinuationToken
+      NextContinuationToken = result.isTruncated ? result.NextContinuationToken : undefined
```

Comment:
Now that we are using `undefined`, can we remove the `isTruncated` test?

Reply:
are you sure it's always undefined ? 
 I would only remove it if the test was using a loose comparison.

Reply:
Nope, I'm not sure, but that's what I understand from the doc (“NextContinuationToken is sent when the result is truncated”).

Reply:
I prefer explicit code (we are sure it's undefined or not) instead of code where you need to check the documentation to be sure. Even using undefined as "not initialized" value is already enough implicit for me